hack/jenkins/e2e.sh

```diff
@@ -326,6 +326,13 @@ if [[ "${E2E_UP,,}" == "true" || "${JENKINS_FORCE_GET_TARS:-}" =~ ^[yY]$ ]]; the
             bucket="${varr[0]}"
             githash="${varr[1]}"
             echo "$bucket / $githash"
+        elif [[ ! -z ${JENKINS_USE_SERVER_VERSION:-} ]]; then
```

Comment:
Doesn't exist (we removed previous use), but if it's not defined, this branch will be skipped.